exclude_files.txt

```diff
@@ -0,0 +1,2 @@
+.git*
```

Comment:
please remove this file

Reply:
Done in https://github.com/splunk-soar-connectors/bitdefendergravityzone/pull/1/commits/c70345d7f8cbbfd5163b29b2d91ea708a9ac7415